### DISCUSSION TITLE
How to Install Dependencies Locally When Using `vcpkg.json`?

### DISCUSSION THREAD

**Discussion by elibroftw** [microsoft/vcpkg#22027] [Q&A]
I tried `set(VCPKG_MANIFEST_MODE OFF)` but that gives me
```
[cmake] CMake Error at C:/Users/maste/Documents/GitHub/vcpkg/scripts/buildsystems/vcpkg.cmake:788 (_find_package):
[cmake]   Could not find a package configuration file provided by "cpr" with any of
[cmake]   the following names:
[cmake] 
[cmake]     cprConfig.cmake
[cmake]     cpr-config.cmake
[cmake] 
[cmake]   Add the installation prefix of "cpr" to CMAKE_PREFIX_PATH or set "cpr_DIR"
[cmake]   to a directory containing one of the above files.  If "cpr" provides a
[cmake]   separate development package or SDK, be sure it has been installed.
```
instead of installing the dependencies at the local user level.

So my question is, how do I make `vcpkg` install libraries at the local level while using a manifest file?

**Comment by Osyotr** [comment]
Just copy or replace contents of `vcpkg/installed/your-triplet` with produced `vcpkg_installed/your-triplet`

**Reply by elibroftw** [reply]
That's just using a respone file with extra steps. I want the cmake tool chain to install dependencies automatically at the system level. Is there a way to do that? Possibly even calling vcpkg with a process?

**Reply by autoantwort** [reply]
> I want the cmake tool chain to install dependencies automatically at the system level.

Not possible, you can use manifest mode or install them manually

**Reply by elibroftw** [reply]
Okay thanks for saving me time. I had made a video using response files and a redditor was hell bent that manifest files could do what I wanted.

Are there plans for giving users the option for vcpkg to install system wide rather than project directory when using manifest files?

**Reply by autoantwort** [reply]
No, and because of the goal of manifest mode this will never happen.

**Reply by elibroftw** [reply]
That's fine. Thanks anyways.
